Odata-docs/support/support-policy.md

```diff
@@ -0,0 +1,127 @@
+---
+title:  "OData libraries support policy"
+description: Support policy of the official OData libraries by Microsoft.
+date:   2024-01-23
+ms.date: 1/23/2023
+author: habbes
+ms.author: clhabins
+---
+
+# Support policy for official .NET OData libraries
```

Comment:
I know internally we've been referring to this as a support policy, but I don't really think it is. This is specifying how we are going to transition from having no support policy, to actually having one. This document is a "one-time" operation, while a policy implies the pattern that will be followed going forward.

Reply:
That's a good call out. Indeed the only "policy" part of this document are the statements mentioning the minimum notice period before EoL or maintenance mode. Once the framework on tracking .NET releases is finalized and added, it would be a proper support policy. I can rename this doc to something else until then (e.g. `Official support for .NET OData libraries` or `Officially supported .NET OData libraries`). What do you think?

Reply:
But I also wanted to point out that in .NET and .NET Core the support policy doc contains both sets of information in the same article: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core. It lists supported and out-of-support versions and their respective timelines. But it also documents the policy behind servicing, release cadence, release types, etc. We could also consider having both in the same "Support policy" doc.

Reply:
> But I also wanted to point out that in .NET and .NET Core the support policy doc contains both sets of information in the same article: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core. It lists supported and out-of-support versions and their respective timelines. But it also documents the policy behind servicing, release cadence, release types, etc. We could also consider having both in the same "Support policy" doc.

Gotcha, as long as we have no issue updating this mage and maybe reordering some sections in the future, I have no problem have a single page contain all of that information similar to .NET